src/js/letters/containers/AddressSection.jsx

```diff
@@ -55,6 +66,9 @@ export class AddressSection extends React.Component {
     }
   }
 
+  componentDidMount() {
+    focusElement('h1');
```

Comment:
Had a really great conversation with an accessibility guru, who mentioned that it may be more intuitive to not steal focus or focus on top of page for the first page of the app, because focusing on H1 may give the false impression that there is no content preceding this element. He mentioned that voice assist users generally understand the first tabbable element on the page is the 'skip to content' link, which he recommended we use to go to the h1 instead of auto-focusing it on page load.

Reply:
Good to know! I'll remove this.